R/plot_allcurves.R

```diff
@@ -0,0 +1,86 @@
+#' Plot All Fitted Curves From tcplfit2_core
+#'
+#' This function takes in output from `tcplfit2_core` and plots
+#' the observed concentration response data with all fitted curves.
+#' The function returns a ggplot object that is capable of
+#' taking additional layers of ggplot add-ons.
+#'
+#' @param output Output from `concRespCore` or `tcplhit2_core`, containing information
+#' about the winning curve fit of a compound.
+#'
+#' @param conc Vector of concentrations (not in log units).
+#'
+#' @param resp Vector of corresponding responses.
+#'
+#' @param log_conc Logical argument. If `TRUE`, convert the concentration (x-axis)
+#' into log-10 scale. Defaults to `FALSE`.
+#'
+#' @return A ggplot plot of the observed concentration response data and
```

Comment:
Suggested rewording: "A ggplot object of the observed concentration response data and all resulting curve fits from `tcplfit2_core`. (Note: The constant model is not included, and only the successful fits will be displayed.)"